setup.py

```diff
@@ -27,41 +27,41 @@
         ),
     ],
     install_requires=[
-        "h5py",
-        "microdf_python",
-        "policyengine-core==1.10.18",
-        "pandas",
-        "pathlib",
-        "pytest",
-        "pytest-dependency",
-        "pyyaml",
-        "requests",
-        "synthimpute",
-        "tables",
-        "tqdm",
-        "click>=8.0.0",
-        "tabulate",
+        "click==8.1.3",
```

Comment:
I'd favour pinning `<NEXT MAJOR` or `<NEXT MINOR`, given that `pip` will automatically find these versions?

Reply:
What's the downside of pinning to patch versions? We had one issue from not pinning to the patch in policyengine-core and it seems like in general more specific environments will improve stability. @ibushong

Reply:
I'm on the road so it's tough for me to see the whole context here. But yeah generally you want to use inequality operators to specify and upper or lower limits of versions to prevent any incompatibilities. Note that this is for setup.py.

I may have miscommunicated my thoughts on this, since the use cases are different between setup.py and requirements.txt

Reply:
Per our conversation, we'll pin everything and check in in a couple weeks to see how much we'd change